docs/content/release-process.md

```diff
@@ -1,76 +1,100 @@
-   created in the [release action] that has been triggered by the tag push.
+Follow the [wrapping up](#wrapping-up) section to finish off the release process. 
+
+## Patch release
+
+To cut a patch release of Spin, you will need to do the following:
+
+1. Backport the commits you wish to include to the release branch you're creating the patch release for. **NOTE** Use the [backport script](https://github.com/fermyon/spin/blob/main/.github/gh-backport.sh) to do so.
+
+1. Switch to the release branch locally and update versions (e.g. `1.1.0-pre0` could `1.1.1`).
+   - Bump the version in Spin's `Cargo.toml`
+   - Update SDK_VERSION in `templates/Makefile`
+   - Run `make build` so that `Cargo.lock` and other associated files are updated
+
+   PR these changes to the release branch ensuring that pull request has a base corresponding to the release branch (e.g. `v1.1`).
+
+1. Create a new tag with a `v` and then the version number, e.g. `v1.1.1`. Then, push the tag to the `fermyon/spin` origin repo.
+
+As an example, via the `git` CLI:
+
+```
+# Switch to the release branch
+git checkout v1.1
+git pull
+
+# Create a GPG-signed and annotated tag
+git tag -s -m "Spin v1.1.1" v1.1.1
+
+# Push the tag to the remote corresponding to fermyon/spin (here 'origin')
+git push origin v1.1.1
+```
+
+**NOTE**
+> The Go SDK tag associated with this release (e.g. `sdk/go/v1.1.0`) will be
+> created in the [release action] that has been triggered by the tag push.
+>
+> When the [release action] completes, binary artifacts and checksums will be
+> automatically uploaded to the GitHub release.
+
+Follow the [wrapping up](#wrapping-up) section to finish off the release process. 
 
-1. When the [release action] completes, binary artifacts and checksums will be
-   automatically uploaded to the GitHub release.
+## Wrapping up
 
-1. A Pull Request will also be created by `fermybot` containing changes to the
-   templates per the updated SDK version. If this is a pre-release for a
-   major/minor version, be sure to change the base of the PR from `main` to the
-   release branch, e.g. `v1.1`. Once CI completes, approve this PR and merge
-   via a merge commit (rather than squash or rebase).
+1. A Pull Request will also be created by `fermybot` containing changes to the templates per the updated SDK version. If this is a pre-release for a major/minor version, be sure to change the base of the PR from `main` to the release branch, e.g. `v1.1`. Once CI completes, approve this PR and merge via a merge commit (rather than squash or rebase).
    
-   This will trigger the `push-templates-tag` job in the [release action],
-   pushing the `spin/templates/v0.9` tag. (Note that this tag may be
-   force-pushed for all patch releases of a given minor release.)
+This will trigger the `push-templates-tag` job in the [release action], pushing the `spin/templates/v0.9` tag. (Note that this tag may be force-pushed for all patch releases of a given minor release.)
```

Comment:
Just a note to mention that the numbering is affected by this line -- could be fixed by an indent?

<img width="1075" alt="Screenshot 2023-08-15 at 10 51 47 AM" src="https://github.com/fermyon/spin/assets/9322017/462a9a78-210c-40eb-870c-7bce346bc8b9">